### PR TITLE
Add support for reading from multiple REDCap projects

### PR DESCRIPTION
**Pull request by mbentz-uf**
**What does this change do?** 

Adds a [combine_adrc_projects.py](./nacc/combine_adrc_projects.py) script.

**Why was this change made?** 

To support reading from multiple projects since the NACC forms have been broken out into multiple REDCap projects.

## Verification

1. Import four [projects xml](https://drive.google.com/drive/folders/1z5bQMbvym7_rHhiBKg1pepo5gzz1tV4w) into your local REDcap instance
2. Create api tokens for each project
3. Create a copy of the config file: `cp nacculator_cfg.ini.example example.config.ini`
4. Update `redcap_server` with your local redcap server (`http://localhost:<port>/api/`)
5. Add the four api tokens to tokens (do not include spaces after the comma) to `tokens` in `example.config.ini`
6. Enable the virtual environment: `source .venv/bin/activate`
7.  Install dependencies: `pip install .`
8. Run the script: `python nacc/combine_adrc_projects.py example.config.ini`

Verify that a `redcap_input.csv` file was generated in the root project directory that contains the records for all projects.

## Affirmations

All of these should have a check by them. Any exception requires an explanation.

* [x] I matched the style of the existing code.
* [x] I added and updated any relevant documentation (inline, `README`, `CHANGELOG`, and such).
* [x] I used Python's type hinting.
* [x] I ran the automated tests and ensured they **ALL** passed.
* [x] I ran the linter and ensured my changes have not introduced **ANY** warnings or errors.
* [x] I have made an effort to minimize code changes and have not included any cruft (preference files, *.pyc files, old comments, print-debugging, unused variables).
* [x] I have made an effort maintain a clear commit history (haven't merged other branches or rebased improperly).
* [x] I have written the code myself or have given credit where credit is due.

### CODE REVIEW

File: nacculator_cfg.ini.example [hunk 1]
```diff
@@ -1,8 +1,9 @@
 [DEFAULT]
 
 [pycap]
+redcap_server: http://<redcap_server>/api/
 token: Your REDCAP Token
-redcap_server: Your Redcap Server
+tokens: token1,token2,token3,token3
```
Comment (s-emerson) [inline nacculator_cfg.ini.example:6]: in order to make less work / confusion for those in the community who already use nacculator, i would  just leave it as "token" with a comment on the previous line about how to fill it out with multiple tokens.

**Comment by s-emerson** [comment]
the code works and creates a file called "redcap_input.csv" in the root folder of nacculator once you have added the necessary new row "tokens" (rather than "token") to your config file.

the branch needs a change to the functionality before i can merge and create a release:
there needs to be no change to nacculator's current workflow, so this can't be a separate script from `run_filters.py`. the combining functionality needs to be within run_filters.py so that nacculator can generate the `redcap_input.csv` file in the `run_CURRENT-DATE` folder, then run the filters on that file automatically. 

i think moving `combine_adrc_projects.py` into a function in `run_filters.py` will fix any other issues (such as putting raw data in the root folder of nacculator instead of in a designated run folder). thank you for working on this!

**Comment by mbentz-uf** [comment]
Changes have been made

**Comment by s-emerson** [comment]
this works great. i tested it locally and in production, tested nacculator's functionality with the downloaded / filtered files, and made sure all the unit tests pass. thank you michael.

**Pull request merged**